src/store/model-store.js

```diff
@@ -31,4 +34,8 @@ store.registerHandler('NO_MODEL_FOUND', () => {
     store.emitChange();
 });
 
+store.registerHandler('MODELS_RECEIVED', data => {
+    model = data.models;
```

Comment:
Is it best to reuse the `model` variable here, or should we make a separate one such as `models` to avoid confusion?